src/DrawTogether.UI/Client/Components/Curve.razor

```diff
@@ -11,16 +13,26 @@
     [Parameter]
     public string Stroke { get; set; } = "#ff0000";
 
-    protected static string PathData(List<Point> Points)
+    protected override bool ShouldRender()
+    {
+        if (_points.Count() != Points.Count)
+        {
+            _points = Points.ToArray();
```

Comment:
Nice, so you're basically using this as a dirty bit? That is clever!

Reply:
Yeah, that was the idea. :)